app/tree/test/stores/TwksStoreSpec.scala

```diff
@@ -8,10 +8,10 @@ import io.github.tetherlessworld.twxplore.lib.tree.stores.TwksStore
 import org.scalatest.{Matchers, WordSpec}
```

Comment:
Shouldn't commit back end changes in a front end branch.

Reply:
I made those back end changes for the branch to pass the CI test. What do you suggest I do instead?

Reply:
It's fine if the backend CI job (scala) doesn't pass, as long as the front end (ts) does.